src/app/pages/camera.cpp

```diff
@@ -582,10 +582,12 @@ void CameraPage::connect_local_stream()
     qDebug() << "camera status: " << this->local_cam->status();
 
     QSize res = this->choose_video_resolution();
+    
+    qDebug() << "RES=" << res;
 
     DASH_LOG(info) << "[CameraPage] Creating GStreamer pipeline with " << this->config->get_cam_local_device().toStdString();
     std::string pipeline = "v4l2src device=" + this->config->get_cam_local_device().toStdString() +
-                           " ! capsfilter caps=\"video/x-raw,width=" + std::to_string(res.width()) + ",height=" + std::to_string(res.height()) + ";image/jpeg,width=" + std::to_string(res.width()) + ",height=" + std::to_string(res.height()) + "\"" +
+                           " ! capsfilter caps=\"video/x-raw,width=" + std::to_string(res.width()) + ",height=" + std::to_string(res.height()) + ",colorimetry=bt601;image/jpeg,width=" + std::to_string(res.width()) + ",height=" + std::to_string(res.height()) + "\"" +
```

Comment:
does that break the pipeline for other cameras? we'll want to add something similar to `this->config->get_cam_local_device()` (and maybe default it to an empty string?) if that's the case

Reply:
Idk if it breaks or if it works. For my camera it works. Someone gotta test this out. But I think it would be perfect to put it in the config like you said.